### PR TITLE
Gradleize

### PR DESCRIPTION
**Pull request by carltonwhitehead**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -18,3 +18,18 @@ local.properties
...
+*.iml
+*.ipr
+*.iws
+classes
+gen-external-apklibs
+
+# Gradle
+.gradle
+build
+gradle
```
Comment (jbarr21) [inline .gitignore]: these last 3 items /should/ be checked into the repo. that is how the gradle wrapper is managed

Comment (carltonwhitehead) [inline .gitignore]: I copied them out of the JazzyListView .gitignore. Should I back these lines out, check in whatever my git is then finding changed, and then re-add these .gitignore lines?

Comment (jbarr21) [inline .gitignore]: yeah, my bad, if u wanna submit a pull against JLV, that would be great =] i think HelloAndroidStudio project would have a better .gitignore to look at

**Comment by jbarr21** [comment]
+1

**Pull request merged**